lucene/core/src/java/org/apache/lucene/search/SortField.java

```diff
@@ -138,7 +146,7 @@ public Provider() {
     }
 
     @Override
-    public SortField readSortField(DataInput in) throws IOException {
+    public SortOrder readSortField(DataInput in) throws IOException {
```

Comment:
Hmm, if `SortOrder` is not (necessarily) serializable, and `SortField` is, shouldn't this return a `SortField`?